src/theme/components/searchBar.ts

```diff
@@ -21,6 +22,14 @@ const SearchBar = defineMultiStyleConfig({
           " > span": {
             display: "none",
           },
+          "> svg": {
+            ...size["medium"],
+          },
```

Comment:
Interesting. This really is just a cleanup and better execution of the container query styles. Nice.

Reply:
Yes! I noticed we were doing some styling with container queries, some with media queries so the radius was changing at a slightly different point than everything else.